docs/docsite/rst/porting_guides/porting_guide_8.rst

```diff
@@ -17,82 +17,91 @@ Ansible 8 is based on Ansible-core 2.15.
 
 We suggest you read this page along with the `Ansible 8 Changelog <https://github.com/ansible-community/ansible-build-data/blob/main/8/CHANGELOG-v8.rst>`_ to understand what updates you may need to make.
 
+Porting Guide for v8.2.0
+========================
 
-Playbook
-========
+Known Issues
+------------
 
-No notable changes
+Ansible-core
+~~~~~~~~~~~~
 
+- ansible-test - The Fedora 37 remote is known to occasionally hang during boot. It is no longer routinely tested as a result. If possible, use the Fedora 38 remote instead.
 
```

Comment:
See the `ansible-core 2.15 Porting Guide <https://docs.ansible.com/ansible-core/2.15/porting_guides/porting_guide_core_2.15.html>`_ for complete details.